packages/kbn-optimizer/limits.yml

```diff
@@ -119,7 +119,7 @@ pageLoadAssetSize:
   observabilityAiAssistantManagement: 19279
   observabilityLogsExplorer: 46650
   observabilityOnboarding: 19573
-  observabilityShared: 80000
+  observabilityShared: 111036
```

Comment:
I wonder if there is anything we could do to avoid increasing the plugin size here.

Reply:
Yeah, I was thinking about this too. But this is out of the scope of this PR. I think this is a good topic for the `Devex-ui-wg`.